Resources/views/Menu/sonata_menu.html.twig

```diff
@@ -41,9 +41,16 @@
         <a href="#">
             {% set translation_domain = item.extra('label_catalogue') %}
             {% set icon = item.extra('icon')|default('') %}
+            {% set spanElement_parent = parent() %}
```

Comment:
No. Variable names and block names has underscores in the project. But this block has name `spanElement`. Variable name `spanElement_parent` means "parent of spanElement block". But I can put a name `parent_content` or like this.